requirements_test.txt

```diff
@@ -5,9 +5,8 @@ pytest-cov
 # numba and hdbscan and numpy internal API
 
 # tridesclous
-numpy==1.22
-Cython
-numba==0.55.2
+numpy==1.21
```

Comment:
why ?

Reply:
a bunch of packages failing to install unfortunately. Let's wait some more (unless you have time to debug it ;) )